cfme/containers/project.py

```diff
@@ -36,17 +35,39 @@ def __init__(self, name, provider):
         self.name = name
         self.provider = provider
 
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div//h1[contains(., "{} (Summary)")]'.format(self.name))
+
     def load_details(self, refresh=False):
         if not self._on_detail_page():
             self.navigate(detail=True)
         elif refresh:
             tb.refresh()
 
+    def click_element(self, *ident):
```

Comment:
@mfalesni @jkrocil  - there are lots of these new functions that do the same thing, do we need these, shoudl they be incorporated elsewhere, anything in the latest hawkular PR that ties in?